### DISCUSSION TITLE
Install pkg.pr.new - Preview Workflow

### DISCUSSION THREAD

**Discussion by birkskyum** [maplibre/maplibre-gl-js#4904] [Ideas]
This Github App https://pkg.pr.new/ , adds a workflow that caches a build, so that each PR and commit becomes installable by npm.

This means that it would be possible to try out changes of a PR by opening a link to a codepen / stackblitz, instead of the current workflow where it's always necessary to checkout the PR, and make a local build / `npm run start`.

**Comment by HarelM** [comment]
Would be nice not to need to install an app for that if possible.
Also, I'm not sure how this will work with jsbin as I think the link is a npm style repo where you need to "download" the package and install it locally.
In theory, I think we can use npm to host PR builds and non released or pre-released version as some other packages are doing, see here for example - I think this is a nightly build, but configuring a version that has the PR number should be fairly easy I guess.
https://www.npmjs.com/package/typescript?activeTab=versions
https://www.npmjs.com/package/react?activeTab=versions
![image](https://github.com/user-attachments/assets/ceb2f1ba-faf9-434c-bd3e-d82a12583a0c)
